pilot/pkg/networking/core/envoyfilter/rc_patch.go

```diff
@@ -310,6 +301,16 @@ func routeConfigurationMatch(patchContext networking.EnvoyFilter_PatchContext, r
 	return true
 }
 
+func getListenerPortFromName(name string) int {
```

Comment:
the logic here is different with https://github.com/istio/istio/blob/dda3b0e343a6319d7f08cf3036ba8d5e9c5ec0a4/pilot/pkg/networking/core/httproute.go#L218, they seem to be the same? (not very sure and not relevant to this PR)